### ISSUE TITLE
M> Dutch Version of Lerner & Nash Foster interview transcript

### ISSUE THREAD

**Issue by Tonyprisca13** [rchain/bounties#283]
Goal: a [DUTCH.srt file](https://docs.google.com/document/d/105HIluXMni6iLST2fX2KLe7CVUULpN0Ct2USzqiqkIQ/edit)
stored in /RChain/Members/Marketing-Community Dev/YouTube
with a translated transcript in Dutch 
Time: 3 week
Budget: $5000

**Comment by Tonyprisca13** [comment]
there is d link for Dutch translation: https://docs.google.com/document/d/105HIluXMni6iLST2fX2KLe7CVUULpN0Ct2USzqiqkIQ/edit?usp=drivesdk&ouid=114592096168227623218

**Comment by BelovedAquila** [comment]
@Tonyprisca13 there is a need to work on your first comment, the links doesn't link.

**Comment by lapin7** [comment]
The doc has been translated with google translate. 
Er klopt geen bal van!!! Zet er maar een andere vertaler op. Dit wordt niks.

**Comment by Tonyprisca13** [comment]
@lapin7  am working with pmoorman on review of the doc.

**Comment by dcpnlau** [comment]
I'm happy to help here and can divide and conquer with whoever is working on this already. If someone can let me know the latest status and send through the doc, I can spend a few hours this week.

**Comment by pmoorman** [comment]
@dcpnlau I spoke with @Tonyprisca13 yesterday on Discord. He can't complete this issue, because he doesn't speak Dutch.

So: _**Right now, nobody is working on this issue.**_

**Comment by Tonyprisca13** [comment]
@pmoorman am still on d issue got some Dutch friends who are will to help

**Comment by Tonyprisca13** [comment]
@dcpnlau presently working on d review of doc

**Comment by Tonyprisca13** [comment]
@dcpnlau  almost done. i will send you the file to review and edit where neccessary.

**Comment by dcpnlau** [comment]
OK thanks. I'll see if I can make some time to make a start on this.

**Comment by dcpnlau** [comment]
Alright sounds good. Sorry - only saw the other messages now. 

Sent on the go

**Comment by peetklasen** [comment]
I have some time too, to contribute. Is it something we can do together @dcpnlau ?

**Comment by Tonyprisca13** [comment]
please send me ur email@dcpnlau

**Comment by peetklasen** [comment]
Is this project almost finished @tonyprisca13 , or is help still needed to come to a result?

**Comment by dcpnlau** [comment]
Hi @peetklasen, it was sent to me. Happy to forward to you and we can divide and conquer. Can you send me your email address? I’m dcpnlau on Discord. 

Sent on the go

**Comment by peetklasen** [comment]
My email address is peetklasen@gmail.com Good idea to collaborate and divide the job. Are you Dutch? ( I am)

**Comment by peetklasen** [comment]
@dcnplau My email address is peetklasen@gmail.com Great idea to collaborate and divide the job. Are you Dutch? ( I am)

**Comment by peetklasen** [comment]
I'm working on the document together with @dcpnlau Can I be assigned?

**Comment by Tonyprisca13** [comment]
@peetklasen Great!

**Comment by peetklasen** [comment]
So: just to get things clear: the document has been automatically translated with Google translate. 
I guess @tonyprisca13 did this job.
The actual translation has still to be done then. 
The actual translation from Google translate Dutch to real Dutch (I mean Dutch that makes sense) will be done bij @dcpnlau and me. Afterwards @dcpnlau and me will review each others translations.

**Comment by peetklasen** [comment]
Just to get things clear: the original document has been automatically translated with Google translate.  I guess @tonypricsa13 did this job. 
Afterwards this document has to be translated to real Dutch (I mean Dutch that makes sense). This job will be done by @dcpnlau and me. Afterwards dcpnlau and me will review each others translations.

**Comment by Tonyprisca13** [comment]
@peetklasen writes:
> I am working on the translation and will start with lines 800 - 1625. @dcnplau will translate lines 1 - 800. Next we will review each others translations. Until now I have translated lines 800 - 1100 (first draft). Tomorrow I will update the progress.
Good work! Do let me know if you will be needing any assistance from me.

**Comment by peetklasen** [comment]
Lines 800 - 1500 have been translated (in draft). For these lines I have now also added the English transcript. This makes reviewing easier. https://docs.google.com/document/d/105HIluXMni6iLST2fX2KLe7CVUULpN0Ct2USzqiqkIQ/edit

**Comment by peetklasen** [comment]
@dcplau has translated lines 1-70. She does not have enough time to finish her part of the translation. We have agreed that I will also translate lines 70 - 800.
I'll keep you informed about the progress.

**Comment by lapin7** [comment]
Mooi zo. Ga zo door :-)

Dan kun je de rewards ook aanpassen voor 28 Feb.
https://docs.google.com/spreadsheets/d/1uxuxx8YN17KCIWcH1cUoGuSm2hAnIya2iAc6wxoaq1o/edit#gid=294454559&range=C2636:N2643

283 |   | M> Dutch Version of Lerner & Nash Foster interview transcript | Marketing | 3 | OPEN | $ 2,000.00 | -50.00 | 1,076.32 | 2,050.00 | Member | Team + Pers Reduce %
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
283 |   | M> Dutch Version of Lerner & Nash Foster interview transcript | Marketing |   | OPEN | $ 2,000.00 |   | 131.26 | 250.00 | Tonyprisca13 | 12.50%
283 |   | M> Dutch Version of Lerner & Nash Foster interview transcript | Marketing |   | OPEN | $ 2,000.00 |   | 0.00 | 0.00 | BelovedAquila | 0.00%
283 |   | M> Dutch Version of Lerner & Nash Foster interview transcript | Marketing |   | OPEN | $ 2,000.00 |   | 0.00 | 0.00 | lapin7 | 0.00%
283 |   | M> Dutch Version of Lerner & Nash Foster interview transcript | Marketing |   | OPEN | $ 2,000.00 |   | 446.28 | 850.00 | dcpnlau | 42.50%
283 |   | M> Dutch Version of Lerner & Nash Foster interview transcript | Marketing |   | OPEN | $ 2,000.00 |   | 446.28 | 850.00 | peetklasen | 42.50%
283 |   | M> Dutch Version of Lerner & Nash Foster interview transcript | Marketing |   | OPEN | $ 2,000.00 |   | 0.00 | 0.00 | kaka56 | 0.00%
283 |   | M> Dutch Version of Lerner & Nash Foster interview transcript | Marketing |   | OPEN | $ 2,000.00 |   | 52.50 | 100.00 | pmoorman | 5.00%

**Comment by peetklasen** [comment]
I made some adjustments to the rewards. I am not so used to the voting system. Hope it is OK like this.
I have translated lines 70 - 500 and 800 - 1625 and will translate lines 500 - 800 in the next few days.

**Comment by peetklasen** [comment]
The job reaches completion  It looks as if @BelovedAquila also had a role in the process. To me it is not clear what this role is or has been.

**Comment by lapin7** [comment]
@BelovedAquila is indeed listed as budget-setter and as reward-setter. However he doesn't get a reward %. So it's fine. If he is claiming a reward, then you can down vote him with a negative percentage. The collaborators have each the power of carrots and sticks so to speak.

2,000.00 | Member | 100.00%
-- | -- | --
275.00 | Tonyprisca13 | 13.75%
0.00 | BelovedAquila | 0.00%
0.00 | lapin7 | 0.00%
225.00 | dcpnlau | 11.25%
1,500.00 | peetklasen | 75.00%
0.00 | kaka56 | 0.00%
0.00 | pmoorman | 0.00%
0.00 | dckc | 0.00%

**Comment by Tonyprisca13** [comment]
@peetklasen I did an adjustment on it please check it up

**Comment by peetklasen** [comment]
I have translated lines 70 to 1625 from the english transcript to Dutch in draft. I have reviewed lines 1-70 that had been translated by @dcnplau. There are still some lines that I have to check / improve. These are marked red. I will also make a comparison with the German translation. That is a nice translation and this can be helpful to improve the current work. I think I can change the status of the document to "final" within a few days. Comments are welcome.
